packages/next-transpile-modules/index.js

```diff
@@ -0,0 +1,41 @@
+module.exports = (nextConfig = {}) => {
+
+  const { transpileModules = [] } = nextConfig
+  const includes = transpileModules.map(module => (new RegExp(`${module}(?!.*node_modules)`)))
+  const excludes = transpileModules.map(module => (new RegExp(`node_modules(?!\/${module}(?!.*node_modules))`)))
+
+  return Object.assign({}, nextConfig, {
+    webpack(config, options) {
+      if (!options.defaultLoaders) {
+        throw new Error(
+          'This plugin is not compatible with Next.js versions below 5.0.0 https://err.sh/next-plugins/upgrade'
+        )
+      }
+
+      config.resolve.symlinks = false
```

Comment:
If there are no modules specified we could just return to avoid adding anything, maybe with a log... or error?